src/sentry/tasks/collect_project_platforms.py

```diff
@@ -39,5 +39,5 @@ def collect_project_platforms(**kwargs):
 
     # remove (likely) unused platform associations
     ProjectPlatform.objects.filter(
-        last_seen__lte=now - timedelta(days=30),
+        last_seen__lte=now - timedelta(days=90),
```

Comment:
Ooooh, for the deletion. I was thinking from an analytics team POV. I figured since this has been running forever, it wasn't hurting anything.

I'll open up the PR to add the index.